includes/class-fontawesome.php

```diff
@@ -1675,6 +1676,45 @@ public function register( $client_preferences ) {
 		$this->client_preferences[ $client_preferences['name'] ] = $client_preferences;
 	}
 
+	/**
+	 * Allows direct querying of the Font Awesome GraphQL metadata API.
+	 * It accepts a GraphQL query string like 'query={versions}'.
+	 * More information about using GraphQL queries can be found {@link https://graphql.org/learn/ here}.
```

Comment:
Great. I think we should also add a link to where a developer could go to read about or experiment with our particular GraphQL schema, once we have some place to send them.

We do have built-in documentation if they just use something like the GraphiQL on the desktop and connect to api.fontawesome.com. So maybe we also suggest here that they could do that to experiment with the API and build a query string that gives them the results they're looking for.